keps/sig-scheduling/20200114-multi-scheduling-profiles.md

```diff
@@ -244,19 +246,32 @@ the scheduler documentation.
 
 ### Test Plan
 
-TODO
+The following tests need to be in place:
 
-### Graduation Criteria
+- **Unit Tests**: For Component Config API conversion and defaults, core scheduler and scheduler
+instantiation. Current tests that use a default scheduler (or default framework) should continue
+passing with minimal changes.
```

Comment:
> Current tests that use a default scheduler (or default framework) should continue passing with minimal changes.

Can you clarify what you mean by "minimal changes"? Do you mean internal wiring changes or external flag/config-driven changes? From a user's perspective, I would expect current behavior to keep working with no changes.

Reply:
I was referring to the test implementation itself. The configuration should have no changes at all. Maybe I should just leave out implementation details?

Reply:
yeah, describe *what* is being tested, not specifically that we will change lines of code in the existing unit tests

Reply:
Sg. I clarified that configuration should have no changes.